### PR TITLE
Testing PostgreSQL version 14

### PR DESCRIPTION
**Pull request by nathanl79**
## Background

Testing the PostgreSQL version 13 on aws, in the /test infrastructure within this pull requests

This PR is not intended to be merged to main, it's only needed to kick off the CircelCI job to confirm if the DB's version is compatible.

### CODE REVIEW

**Comment by anniehedgpeth** [comment]
/test private-active-active

@nathanl79 Looks like you don't have the right permissions to run it. We'll get that sorted out later, but for now I'll run this.

**Comment by nathanl79** [comment]
> /test private-active-active
> 
> @nathanl79 Looks like you don't have the right permissions to run it. We'll get that sorted out later, but for now I'll run this.

Thank you, Annie! =)

File: tests/private-active-active/main.tf [hunk 1]
```diff
@@ -51,6 +51,7 @@ module "private_active_active" {
   redis_encryption_in_transit = true
   redis_use_password_auth     = true
   tfe_subdomain               = local.test_name
+  database_version            = 13
```
Comment (sudomateo) [inline tests/private-active-active/main.tf:54]: This should be `postgres_engine_version`.

https://github.com/hashicorp/terraform-aws-terraform-enterprise/blob/main/variables.tf#L153-L157

**Review by sudomateo — CHANGES_REQUESTED** [review]

File: tests/private-active-active/main.tf (hunk 1 above)
Comment (nathanl79) [reply tests/private-active-active/main.tf]: I made the change to add `postgres_engine_version = 13`

**Pull request closed without merging**